pkg/network/usm/kafka_monitor_test.go

```diff
@@ -824,13 +824,13 @@ func testKafkaFetchRaw(t *testing.T, tls bool, apiVersion int) {
 			buildResponse: func(topic string) kmsg.FetchResponse {
 				record := makeRecord()
 				var records []kmsg.Record
-				for i := 0; i < 5; i++ {
+				for i := 0; i < 2; i++ {
```

Comment:
nit: it would be nice to use consts, to ease the reading of the test

Reply:
I looked into this, but given that the different tests uses different values and putting the constant definitions close to each test case is not possible given that we're in the middle of an array, I'm not sure if consts will improve things. It could be better to have these two values as fields in the test struct to eliminate the duplication and improve readability. I will leave such a change out of this PR unless there are objections.